clearwater-memcached/usr/share/clearwater/infrastructure/scripts/memcached

```diff
@@ -25,4 +25,6 @@ sed -e 's/^-l .*$/-l '$listen_address'/g'\
     -e 's/^# *-v *$/-v/g'\
     -e 's/^\(# *\|\)-c.*$/-c 4096/g' </etc/memcached.conf >/etc/memcached_11211.conf
 
+echo "\n# Disable listening on UDP\n-U 0" >> /etc/memcached_11211.conf
+
```

Comment:
Hm, interesting - the line worked as expected on my box when running the script. Furthermore, unless I'm misunderstanding the `sed` command above, we're re-creating the `/etc/memcached_11211.conf` script every time we run this - so we shouldn't be adding these lines repeatedly over time?

I'll look at those utilities though - agreed that I should be using them, given that we have them and they are used elsewhere.